contrib/seeds/makeseeds.py

```diff
@@ -30,7 +30,7 @@
 PATTERN_IPV4 = re.compile(r"^((\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})):(\d+)$")
 PATTERN_IPV6 = re.compile(r"^\[([0-9a-z:]+)\]:(\d+)$")
 PATTERN_ONION = re.compile(r"^([abcdefghijklmnopqrstuvwxyz234567]{16}\.onion):(\d+)$")
-PATTERN_AGENT = re.compile(r"^(/Satoshi:0.14.(0|1|2|99)/|/Satoshi:0.15.(0|1|2|99)|/Satoshi:0.16.(0|1|2|99)/)$")
+PATTERN_AGENT = re.compile(r"^(/Satoshi:0.14.(0|1|2|3|99)/|/Satoshi:0.15.(0|1|2|99)|/Satoshi:0.16.(0|1|2|3|99)/|/Satoshi:0.17.(0|1|2|99)/|/Satoshi:0.18.99/)$")
```

Comment:
Should we perhaps exclude nodes vulnerable to CVE-2018-17144?

Reply:
And why is this only including Core nodes in the first place?

Reply:
CVE-2018-17144: for their own protection?